user-guide/rules.md

```diff
@@ -463,67 +465,88 @@ Apart from returning files or predefined responses, a rule can perform the follo
+* If a session is matched with conditions that depend on its response (for example, a response body contains "HTML"), then any final action in any rule that matches the session will be ignored. The reason for this behavior is that final actions replace the response. By design, Fiddler is not intended to replace a response that was already received and matched conditions in a rule.
 
 * Non-final actions will allow other actions from the same rule or different rules with lower priority, which match the session, to execute.
 
 ## Rules Queue
 
-The **Rules Queue**, also the __Queue Viewer__ or __Queue Rules Viewer__, is a queue of the created rules. The **Rules Queue** allows you to enable, disable, promote, denote, edit, export, import, share, and immediately execute the listed rules.
+The **Rules Queue** is a visual tree that lists all active rules & groups in a priority queue. It is important to note that each active rule or group has higher execution priority than other rules & groups placed lower in the queue. The **Rules Queue** allows you to enable, disable, promote, denote, edit, export, import, share, and immediately execute the listed rules.
 
-When using a rule from the **Rules Queue**, take into consideration the following tips:
+To use a rule (or a group of rules) from the **Rules Queue**, take into consideration the following tips:
 
-- The **Rules** tab must be enabled for any rule from the queue to be executed.
-- Only **enabled** rules from the queue will be applied on ongoing Live traffic capture.
-- You can explicitly apply a rule for captured traffic by using the **Play** button (executes the rule on current sessions).
-- The rules in the **Rules Queue** are executed with priority based on their position in the queue. Use **Demote** and **Promote** buttons to change a rule position (and priority) in the queue.
-- Some rules might trigger [a final action](#final-and-non-final-actions), which will stop executing any other rules with lower priority.
+- The **Rules** tab must be active (switch ON).
 
-### Rules Priority
+- Only **active** rules from the queue will be applied on ongoing Live traffic capture. If the rule is contained within a group, then the group must be active as well.
+
+- You can explicitly trigger a rule for already captured (including saved) traffic through the **Execute** button (executes the rule on current sessions).
 
-By default, the **Rules Queue** will list all created and imported rules based on their priority. Rules will be executed according to their appearance, starting from the first. When selecting specific rules and using the explicit **Execute** button, they will run in the order they appear in the queue.
+- The rules in the **Rules Queue** are executed with priority based on their position in the queue. Use the **Demote** and **Promote** buttons to change the queue's rule position (and priority).
```

Comment:
We may also want to mention drag&drop here and maybe in the list of options below?